docs/_specification/1.2-DRAFT/data-entities.md

```diff
@@ -34,25 +34,25 @@ parent: RO-Crate 1.2-DRAFT
 
 The primary purpose for RO-Crate is to gather and describe a set of _Data entities_ in the form of:
 
-* Files
+* Files which are datastreams available on the local file system or over the web
 * Directories
-* Web resources
+
 
 The data entities can be further described by referencing [contextual entities](contextual-entities) such as persons, organizations and publications.
 
 ## Referencing files and folders from the Root Data Entity
 
 Where files and folders are represented as _Data Entities_ in the RO-Crate JSON-LD, these MUST be linked to, either directly or indirectly, from the [Root Data Entity](root-data-entity) using the [hasPart] property. Directory hierarchies MAY be represented with nested [Dataset] _Data Entities_, or the Root Data Entity MAY refer to files anywhere in the hierarchy using [hasPart].
 
-_Data Entities_ representing files MUST have `"File"` as a value for `@type`. `File` is an RO-Crate alias for <http://schema.org/MediaObject>. The term _File_ includes:
--  _Attached_ resources where `@id` is a URI (path) relative to the _RO-Crate Root_ which MUST resolve to a file.
--  _Detached_ "downloadable" resources where `@id` is an absolute URI which resolves to a single datastream that can be downloaded and saved as a file. _Detached_ Files SHOULD NOT reference intermediate resources such as splash-pages, search services or web-based viewer applications.
+_Data Entities_ representing files: MUST have `"File"` as a value for `@type`. `File` is an RO-Crate alias for <http://schema.org/MediaObject>. The term _File_ includes:
+-  _Attached_ resources which are available locally and
+-  _Detached_ "downloadable" resources which can be can be downloaded and saved as a file.
```

Comment:
I quite like the previous explicit inclusion of "Detached Files SHOULD NOT reference intermediate resources such as splash-pages, search services or web-based viewer applications."
Is there a particular reason to get rid of it?

Reply:
No, a mistake - putting it back under web-based data entities section